src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -339,6 +339,8 @@ private static ProjectRestoreMetadata GetMSBuildMetadata(PackageSpec packageSpec
                     GetBoolOrFalse(restoreLockProperties, "restoreLockedMode", packageSpec.FilePath));
             }
 
+            msbuildMetadata.PackagesConfigPath = rawMSBuildMetadata.GetValue<string>("packagesConfigPath");
```

Comment:
🤢

Reply:
is JsonPackageSpecReader only used for project.json projects? If so, I can delete this line. At the time I was just updating anywhere it looked like the dgSpec might be deserialised.

Reply:
This merits more investigation/discussion.

Reply:
I forgot to reply earlier. `JsonPackageSpecReader` is used to read the package spec used by PR projects. It's not specific to `project.json` projects. As long as the design for my work is to reuse the dgspec generation to read MSBuild properties for PC projects, we can reduce the number of times we hit the filesystem trying to determine the packages.config file name.

Reply:
It's used to read project.json & PackageReference cause they are pretty much the same beyond the way you declare things. 

I'll get back to this with another review, but just wanted to clarify things.

Reply:
Resolving conversation as the `PackagesConfigPath` property has been moved into a PC-specific class.